SimPEG/electromagnetics/static/utils/static_utils.py

```diff
@@ -2087,145 +1546,777 @@ def gen_3d_survey_from_2d_lines(
-                    np.ones(100) * locz,
-                    "b--",
-                    lw=0.5,
-                )
-        return ax.plot(resistivity, z, "k-", **kwargs)
 
 
 ############
 # Deprecated
 ############
 
-
-def plot_pseudoSection(
+def plot_pseudosection(
     data,
     ax=None,
     survey_type="dipole-dipole",
-    data_type="appConductivity",
-    space_type="half-space",
+    data_type="apparent conductivity",
+    space_type="half space",
+    plot_type="pcolor",
     clim=None,
     scale="linear",
     sameratio=True,
-    pcolorOpts={},
-    data_location=False,
+    pcolor_opts={},
+    contour_opts={},
+    cbar_opts={},
+    data_locations=False,
     dobs=None,
     dim=2,
+    y_values="n-spacing",
+    **kwargs,
 ):
 
     warnings.warn(
-        "The plot_pseudoSection method has been deprecated. Please use "
-        "plot_pseudosection instead. This will be removed in version"
+        "The plot_pseudosection method has been deprecated and may not work properly!!! "
+        "Please use plot_2d_pseudosection instead. This will be removed in version"
         " 0.15.0 of SimPEG",
         DeprecationWarning,
     )
 
-    return plot_pseudosection(
-        data=data,
-        ax=ax,
-        survey_type=survey_type,
-        data_type=data_type,
-        space_type=space_type,
-        clim=clim,
-        scale=scale,
-        sameratio=sameratio,
-        pcolor_opts=pcolorOpts,
-        data_locations=data_location,
-        dobs=dobs,
-        dim=dim,
+    return plot_2d_pseudosection(data.survey, data.dobs, 'scatter', ax=ax, scale=scale)
+
+def apparent_resistivity(data_object, space_type='half space', dobs=None, eps=1e-10, **kwargs):
+
+    warnings.warn(
+        "The apparent_resistivity method has been deprecated. Please use "
+        "apparent_resistivity_from_voltage instead. This will be removed in version"
+        " 0.15.0 of SimPEG",
+        DeprecationWarning,
     )
 
+    if dobs is None:
+        dobs = data_object.dobs
+
+    return apparent_resistivity_from_voltage(data_object.survey, dobs, space_type=space_type)
 
-def gen_DCIPsurvey(endl, survey_type, a, b, n, dim=3, d2flag="2.5D"):
+
+def source_receiver_midpoints(survey, **kwargs):
     warnings.warn(
-        "The gen_DCIPsurvey method has been deprecated. Please use "
-        "generate_dcip_survey instead. This will be removed in version"
+        "The source_receiver_midpoint method has been deprecated. Please use "
+        "pseudo_locations instead. This will be removed in version"
+        " 0.15.0 of SimPEG",
+        DeprecationWarning,
+    )
+
+    return pseudo_locations(survey, kwargs)
+
+def plot_layer(rho, mesh, **kwargs):
+    warnings.warn(
+        "The plot_layer method has been deprecated. Please use "
+        "plot_1d_layer_model instead. This will be removed in version"
+        " 0.15.0 of SimPEG",
+        DeprecationWarning,
+    )
+
+    return plot_1d_layer_model(mesh.hx, rho)
```

Comment:
again, not passing any of the kwargs on to the underlying function

Reply:
done